sharding/nodesCoordinator/indexHashedNodesCoordinator.go

```diff
@@ -1191,8 +1186,3 @@ func createValidatorInfoFromBody(
 
 	return allValidatorInfo, nil
 }
-
-func (ihnc *indexHashedNodesCoordinator) updateEpochFlags(epoch uint32) {
-	ihnc.flagWaitingListFix.SetValue(epoch >= ihnc.waitingListFixEnableEpoch)
-	log.Debug("indexHashedNodesCoordinator: waiting list fix", "enabled", ihnc.flagWaitingListFix.IsSet())
-}
```

Comment:
if we remove this, it seems like this `flagWaitingListFix` var will not be set anymore, not sure if it safe to remove it

Reply:
the flag was set but never used, that's why I completely removed it

Reply:
nevermind, i've just seen that it was moved back in part 7